template.go

```diff
@@ -102,6 +112,22 @@ func groupByKeys(entries interface{}, key string) ([]string, error) {
 	return ret, nil
 }
 
+// groupByLabel is the same as groupBy but over a given label
+func groupByLabel(entries interface{}, label string) (map[string][]interface{}, error) {
+	getLabel := func(v interface{}) (interface{}, error) {
+		if container, ok := v.(RuntimeContainer); ok {
+			if container.Labels[label] == "" {
```

Comment:
This will break if the label actually has a blank value. You may want to check the second return value from `container.Labels[label]` to see if it is `false` instead of checking for `""`. I did this in the `whereLabel` function.